src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/AzureFileShareModels/AzureFileShareItem.cs

```diff
@@ -62,12 +37,11 @@ public class AzureFileShareItem : ItemBase
         /// <param name="policyName">Name of the protection policy associated with this protected item</param>
         public AzureFileShareItem(ProtectedItemResource protectedItemResource,
             string containerName, ContainerType containerType, string policyName)
-            : base(protectedItemResource, containerName, containerType)
+            : base(protectedItemResource, containerName, containerType, policyName)
         {
             AzureFileshareProtectedItem protectedItem = (AzureFileshareProtectedItem)protectedItemResource.Properties;
             LastBackupStatus = protectedItem.LastBackupStatus;
```

Comment:
Please move the initialization for `LastBackupStatus, LastBackupTime, ProtectionState and ProtectionStatus` to the base class - both in VM and Files.

Reply:
workload specific properties, can't move it to base class